cosmic/src/hrdiag.f

```diff
@@ -800,6 +800,11 @@ SUBROUTINE hrdiag(mass,aj,mt,tm,tn,tscls,lums,GB,zpars,
                         endif
                      endif
                      mt = mrem
+* Store the initial BH mass for calculating the ISCO later
```

Comment:
What do you think about initializing Mbh_initial to 0, then checking if Mbh_initial.eq.0 instead of BHbirth_ind?

Reply:
Yep makes total sense and is definitely cleaner, I’ll push that change to this PR.